app/Http/Resources/V4/StreamingLinksResource.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Resources\V4;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class StreamingLinksResource extends JsonResource
+{
+
+    /**
+     *  @OA\Schema(
+     *      schema="streaming_links",
+     *      description="Streaming links",
+     *
+     *      @OA\Property(
+     *          property="data",
+     *          type="array",
+     *
+     *          @OA\Items(
+     *              type="object",
+     *
+     *              @OA\Property(
+     *                   property="name",
+     *                   type="string",
+     *              ),
+     *              @OA\Property(
+     *                   property="url",
+     *                   type="string",
+     *              ),
+     *          ),
+     *      ),
+     *  )
+     */
+
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
```

Comment:
use type annotation `public function toArray(\Illuminate\Http\Request $request): array`

Reply:
as discussed on discord, let's not try type annotate this one